README.md

```diff
@@ -340,83 +257,32 @@ GraphDB uses logback to configure logging using the `logback.xml` file.
-GraphDB's Helm chart supports preload and LoadRDF tools for preloading data. It also supports Storage tool for scanning and repairing data. There are a few options that are used to run the needed commands.
-
-Those options are described in the subsection `graphdb.tools.*` and they are:
-
-- resources - to set the needed resources in order to run the tools. Bear in mind that if you don't give the init containers enough resources, the tools might fail.
-```bash
-resources:
-  limits:
-    cpu: 4
-    memory: "10Gi"
-  requests:
-    cpu: 4
-    memory: "10Gi"
-```
-- preload - tool to preload data in a chosen repository.
-  - trigger - If trigger is set to true, then the preload tool will be run while initializing the deployment.
-  - flags - options to add to the command. The possible options are "-f", "-p", "-r". If you use the "-f" option, the tool will override the repository and could lose some data.
-  - rdfDataFile - the file that is added in the mounted directory.
-
-For more information about the Preload tool see: https://graphdb.ontotext.com/documentation/enterprise/loading-data-using-preload.html
-
-- loadrdf - tool to preload data in a chosen repository.
-  - trigger - if trigger is set to true, then the loadrdf tool will be run while initializing the deployment.
-  - flags - options to add to the command. The possible options are "-f", "-p". If you use the "-f" option, the tool will override the repository and could lose some data.
-  - rdfDataFile - the file that is added in the mounted directory.
-
-For more information about the LoadRDF tool see: https://graphdb.ontotext.com/documentation/enterprise/loading-data-using-the-loadrdf-tool.html
-
-- storage_tool - tool for scanning and repairing data.
-  - trigger - if trigger is set to true, then the storage tool will be run while initializing the deployment.
-  - command - the command to run the storage-tool with.
-  - repository - repo to run command on.
-  - options - additional options to run the storage-tool with.
-
-For more information about the Storage tool see https://graphdb.ontotext.com/documentation/enterprise/storage-tool.html
-
 ### Networking
 
-By default, GraphDB's Helm chart comes with a default Ingress and also Kong for more flexibility in configuring instances paths.
-Both the Ingress and Kong can be disabled by switching `kong.enabled` and `ingress.enabled`.
+By default, GraphDB's Helm chart comes with a default Ingress.
+The Ingress =can be disabled by switching `ingress.enabled` to false.
 
 ### Cloud deployments specifics
 
 Some cloud kubernetes clusters have some specifics that should be noted. Here are some useful tips on some cloud K8s clusters:
 
 ##### Google cloud
 
-In Google's k8s cluster services, the root directory is not writable. By default GraphDB's chart uses `/data` directory to store instances data.
+In Google's k8s cluster services, the root directory is not writable. By default, GraphDB's chart uses `/data` directory to store instances data.
```

Comment:
not true i guess? we're mounting the whole home right?

Reply:
The /data directory spoken here about is the one set as deployment.storage, for which it is true.